plugin.js

```diff
@@ -61,14 +61,17 @@ function plugin (fastify, options, next) {
     return cookie.parse(cookieHeader, options.parseOptions)
   })
   fastify.decorateRequest('cookies', null)
+  fastify.decorateRequest('unsignCookie', function unsignCookieRequestWrapper (value) {
+    return signer.unsign(value)
+  })
   fastify.decorateReply('setCookie', function setCookieWrapper (name, value, options) {
     return fastifyCookieSetCookie(this, name, value, options, signer)
   })
   fastify.decorateReply('clearCookie', function clearCookieWrapper (name, options) {
     return fastifyCookieClearCookie(this, name, options)
   })
 
-  fastify.decorateReply('unsignCookie', function unsignCookieWrapper (value) {
+  fastify.decorateReply('unsignCookie', function unsignCookieReplyWrapper (value) {
```

Comment:
For the above reason, this function has been renamed to improve descriptiveness, a clear separation of where it occurs, and naming consistency. But I can revert back to `unsignCookieWrapper` for backward compatibility. Perhaps someone expects to see this in the call stack when analyzing the logs (although I strongly doubt it).